src/io/User.java

```diff
@@ -35,7 +46,10 @@ public void setPassword(String password) {
     }
 
     public String getBankCardNumber() {
-        return bankCardNumber;
+        byte [] cardNumber = bankCardNumber.getBytes();
+        Base64.getEncoder().encode(cardNumber);
+        String s = cardNumber.toString();
```

Comment:
Use "Arrays.toString()" instead.

Reply:
But Arrays.toString()  returns us a mass of bytes, but we need to show the data through encrypted Base64․ It is not so?